src/public/sass/UI/product/_product-info.scss

```
@@ -0,0 +1,71 @@
+@use '../../base/variables' as v;
+@use '../../base/mixins' as m;
+
+.product-info {
+  width: 90%;
+  max-width: 120rem;
+  margin: 1rem auto;
+  scroll-margin-top: 11rem;
```

```suggestion
  scroll-margin-top: 13rem;
```
Add more margin on mobile views please